src/site/antora/modules/ROOT/pages/manual/cloud.adoc

```diff
@@ -15,549 +15,267 @@
+While doing so, we will also try to share guides on some popular scenarios.
 
-All the solutions discussed on this page are predicated with the idea that log files cannot permanently
-reside on the file system and that all log events should be routed to one or more log analysis tools that will
-be used for reporting and alerting. There are many ways to forward and collect events to be sent to the
-log analysis tools.
+[#motivation]
+== Motivation
 
-Note that any approach that bypasses Docker's logging drivers requires Log4j's
-https://logging.apache.org/log4j/2.x/manual/lookups.html#DockerLookup[Docker Lookup] to allow Docker attributes to be injected into the log events.
+Most modern software is deployed in https://en.wikipedia.org/wiki/Service-oriented_architecture[service-oriented architectures].
+This is a very broad domain and can be realized in an amazingly large number of ways.
+Nevertheless, they all redefine the notion of an application:
 
-=== Logging to the Standard Output Stream
+* Deployed in *multiple instances*
+* Situated in *multiple locations*; either in the same rack, or in different data centers located in different continents
+* Hosted by *multiple platforms*; hardware, virtual machine, container, etc.
+* *Polyglot*; a product of multiple programming languages
+* *Scaled* on demand; instances come and go in time
 
-As discussed above, this is the recommended 12-Factor approach for applications running in a docker container.
-The Log4j team does not recommend this approach for performance reasons.
+Naturally, logging systems also evolved to accommodate these needs.
+In particular, the old practice of _"monoliths writing logs to files rotated daily"_ has changed in two major angles:
 
-image:DockerStdout.png[Stdout, "Application Logging to the Standard Output Stream"]
+Application delivers logs differently::
 
-=== Logging to the Standard Output Stream with the Docker Fluentd Logging Driver
+Applications no longer write logs to files, but <<structured-encoding,encode them structurally>>, and deliver them to an external system centrally managed.
+Most of the time this is a <<proxy,proxy>> (a library, a sidecar container, etc.) that takes care of discovering the log storage system and determining the right external service to forward the logs to.
 
-Docker provides alternate https://docs.docker.com/config/containers/logging/configure/[logging drivers],
-such as https://docs.docker.com/config/containers/logging/fluentd/[fluentd], that
-can be used to redirect the standard output stream to a log forwarder or log aggregator.
+Platform stores logs differently::
 
-When routing to a log forwarder it is expected that the forwarder will have the same lifetime as the
-application. If the forwarder should fail the management tools would be expected to also terminate
-other containers dependent on the forwarder.
+There is no longer `/var/log/tomcat/app.log` combining all logs of a monolith.
```

Comment:
_Nit_: Most Linux distributions use `/var/log/tomcat/catalina.out` for `stdout/stderr`.